source/docs/hardware/sensors/accelerometers-hardware.rst

```diff
@@ -5,31 +5,42 @@ Accelerometers are common sensors used to measure acceleration.
 
 In principle, precise measurements of acceleration can be double-integrated and used to track position (similarly to how the measurement of turn rate from a gyroscope can be integrated to determine heading) - however, in practice, accelerometers that are available within the legal FRC price range are not nearly accurate for this use.  However, accelerometers are still useful for a number of tasks in FRC.
 
-The roboRIO comes with a built-in three-axis accelerometer that all teams can use, however teams seeking more-precise measurements may purchase and use a peripheral accelerometer, as well.
+The roboRIO comes with a :ref:`built-in three-axis accelerometer <docs/software/sensors/accelerometers-software:BuiltInAccelerometer>` that all teams can use, however teams seeking more-precise measurements may purchase and use a peripheral accelerometer, as well.
 
-Several popular FRC devices (known as "inertial measurement units," or "IMUs") combine both an accelerometer and a gyroscope.  Popular FRC example include:
-
-  - `Analog Devices ADIS16448 and ADIS 16470 IMUs <https://www.analog.com/en/landing-pages/001/first.html>`__
-  - `CTRE Pigeon IMU <https://www.ctr-electronics.com/gadgeteer-imu-module-pigeon.html>`__
-  - `Kauai Labs NavX <https://pdocs.kauailabs.com/navx-mxp/>`__
 
 Types of accelerometers
 -----------------------
 
-There are two types of accelerometers commonly-used in FRC: single-axis accelerometers, and multi-axis accelerometers.
+There are three types of accelerometers commonly-used in FRC: single-axis accelerometers, multi-axis accelerometers, and IMUs.
 
 Single-axis accelerometers
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
```

Comment:
Is there no picture of a single-axis accelerometer?

Reply:
Added in 66f2eae.